src/mode.js

```diff
@@ -60,15 +60,18 @@ export function getMode(opt_win) {
  * @return {!ModeDef}
  */
 function getMode_(win) {
+  const AMP_CONFIG = self.AMP_CONFIG || {};
+
   // Magic constants that are replaced by closure compiler.
   // IS_MINIFIED is always replaced with true when closure compiler is used
   // while IS_DEV is only replaced when `gulp dist` is called without the
   // --fortesting flag.
   const IS_DEV = true;
   const IS_MINIFIED = false;
 
-  const localDevEnabled = !!(self.AMP_CONFIG && self.AMP_CONFIG.localDev);
-  const runningTests = IS_DEV && !!(win.AMP_TEST || win.__karma__);
+  const localDevEnabled = !!self.AMP_CONFIG.localDev;
```

Comment:
Perhaps this line should also use `AMP_CONFIG` instead of `self.AMP_CONFIG`?

Reply:
lol that was the entire reason I created this const in the first place, forgot this line...